assets/stylesheets/pages/_reactivex.scss

```diff
@@ -0,0 +1,11 @@
+._reactivex {
+  @extend %simple;
+
+  img {
+    max-width: 50%;
+  }
+
+  figure {
+    margin: 0;
+  }
+}
```

Comment:
On http://localhost:9292/reactivex/documentation/observable, `<dfn>` elements are rendered with a “?” cursor and a dotted underline, implying that you can hover it to see a definition. On the original site, they’re rendered as plain *italics*.
```suggestion
  dfn {
    cursor: text;
    text-style: italic;
    text-decoration: none;
  }
}
```